operator/v1/types_machineconfiguration.go

```diff
@@ -56,3 +63,41 @@ type MachineConfigurationList struct {
 	// Items contains the items
 	Items []MachineConfiguration `json:"items"`
 }
+
+type ManagedBootImages struct {
+	// machineSets allows granular control of the machine resources that will be enrolled to the ManagedBootImages feature.
+	// +required
```

Comment:
This is not the correct tag
```suggestion
	// +kubebuilder:validation:Required
```